sites/all/modules/features/dccr_ftr_workshop/dccr_ftr_workshop.module.inc

```diff
@@ -96,3 +96,77 @@ function dccr_ftr_workshop_countdown($action, $content_id) {
   $wrapper->save();
 }
 
+/**
+ * Implements hook_cron().
+ */
+function dccr_ftr_workshop_cron() {
+  $week_before = strtotime('-7 day',DCCR_CAMP_START_DATE);
+  $three_days_before = strtotime('-3 day',DCCR_CAMP_START_DATE);
+ if (DCCR_CAMP_START_DATE >= REQUEST_TIME && $week_before <= REQUEST_TIME && !variable_get('REMINDER_WEEK_BEFORE', FALSE)) {
```

Comment:
First condition is not necessary.
variable name should be prefixed with module name and follow variable names conventions (no uppercase)